gadget/install/install.go

```diff
@@ -251,6 +248,16 @@ func createPartitions(model gadget.Model, gadgetRoot, kernelRoot, bootDevice str
 	if err != nil {
 		return "", nil, nil, 0, fmt.Errorf("cannot read %v partitions: %v", bootDevice, err)
 	}
+	// Tweak label for the encryption case.
+	// TODO: try to build LaidOutVolume providing OnDiskVolume so this is not needed.
+	if options.EncryptionType != "" {
+		for i := range laidOutBootVol.LaidOutStructure {
+			switch laidOutBootVol.LaidOutStructure[i].GadgetStructure.Role {
+			case gadget.SystemData, gadget.SystemSave:
+				laidOutBootVol.LaidOutStructure[i].Label += "-enc"
+			}
+		}
+	}
 
```

Comment:
it's a bit hard to tell if this correct or not. We need to think more what to do about Filesystem and Label.